src/mondo_robot_template.py

```diff
@@ -50,13 +56,14 @@ def run(input_file: str = INPUT_FILE, output_file: str = OUTPUT_FILE):
     # - Done by proxy: UID <-> CUI <-> MONDO
     df_medgen_medgenuid = df[df['source'] == 'MedGen'][['source_id', 'xref_id']].rename(
         columns={'source_id': 'medgen_uid'})
-    out_df_uid = pd.merge(df_medgen_mondo, df_medgen_medgenuid, on='xref_id')[['mondo_id', 'medgen_uid']]\
-        .rename(columns={'medgen_uid': 'xref_id'})
+    out_df_uid = pd.merge(df_medgen_mondo, df_medgen_medgenuid, on='xref_id').rename(
+        columns={'xref_id': 'source_id', 'medgen_uid': 'xref_id'})[['mondo_id', 'xref_id', 'source_id']]
```

Comment:
## Basic changes
PR basically involves just changing 3 lines. I had already been doing a JOIN to map `mondo_id` to UIDs. One of those tables being joined, though, already had the UID to source id CUI mappings. So I just needed to change it to not drop that column. Then, I just needed to namespace the CUIs with "UMLS:". Finally, I needed to update the robot sub-header.

Reply:
Re-ran the script afterwards and uploaded the new robot template to the latest [release](https://github.com/monarch-initiative/medgen/releases/tag/2023-09-03).